src/auth/basic/Config.cc

```diff
@@ -36,7 +36,7 @@
 /* Basic Scheme */
 static AUTHSSTATS authenticateBasicStats;
 
-helper *basicauthenticators = nullptr;
+HelperPointer basicauthenticators = nullptr;
```

Comment:
Pointer types should never need explicit ```nullptr``` instantiation.
```suggestion
HelperPointer basicauthenticators;
```